### ISSUE TITLE
Prevent scanning for photos/tracks in specific folders

### ISSUE THREAD

**Issue by johnmcd88** [nextcloud/maps#1324]
**Is your feature request related to a problem? Please describe.**
Would it be possible to limit what folders Maps scans for photos/tracks - something like the '.nomedia' file Android supports? I have multiple external storages in NC that are CIFS/SMB mounts, and some of them contain a lot of picture files such as book covers, etc. that I would prefer Maps not scan since they're not relevant to what I'd like Maps to display. 

**Describe the solution you'd like**
The ability to place a file (something like 'no.index.maps') into a folder and have Maps skip scanning that folder and all subfolders for any pictures or tracks.

**Describe alternatives you've considered**
I haven't been able to find any workarounds.

**Additional context**

**Comment by GAS85** [comment]
#143 It should actually work with `.nomedia`. Did you check it?

**Comment by johnmcd88** [comment]
Thanks - I created a bunch of .nomedia files and I'm running a scan right now.

**Comment by johnmcd88** [comment]
Nope, it doesn't seem to have worked. Maps is still scanning folders that have .nomedia files in them.

**Comment by johnmcd88** [comment]
p.s. I looked at my NAS logs and verified that the scan had actually read the .nomedia files, then proceeded to scan files/subfolders in that same folder.

**Comment by johnmcd88** [comment]
p.p.s. as per another post I found I also added '.m.nomedia' files to each folder and it appears those are also ignored.

**Comment by GAS85** [comment]
Never heard about `.m.nomedia` 🤔

**Comment by johnmcd88** [comment]
It's something I saw on another forum somewhere and I thought it was worth a try.
